samples/subsys/usb/dfu/sample.yaml

```diff
@@ -7,8 +7,8 @@ tests:
       mimxrt1050_evk_qspi mimxrt1020_evk mimxrt1015_evk mimxrt1060_evk
       mimxrt1050_evk mimxrt1060_evk_hyperflash nucleo_f207zg
     depends_on: usb_device
-    filter: dt_compat_enabled_with_label("fixed-partitions", "slot0_partition") and
-            dt_compat_enabled_with_label("fixed-partitions", "slot1_partition") and
+    filter: dt_label_with_parent_compat_enabled("slot0_partition", "fixed-partitions") and
```

Comment:
this is getting ugly with multiple checks, can we shorten those filter names?. no idea how anyone will be able to remember this macro name TBH.

Reply:
What about `dt_child_label_parent_enabled` ?
@mbolivar-nordic would it sounds still correct? If not, any alternative ?

Reply:
SGTM

Reply:
> What about dt_child_label_parent_enabled ?

@nashif Would it be any better ?